### PR TITLE
Update 02-configuration.mdx

### PR DESCRIPTION
**Pull request by milstan**
Milan's additions to detailed configuration tutorial

### CODE REVIEW

**Comment by milstan** [comment]
This is an update jsut to list the stuff that one can configure, and improve coverage. I'll later work on simplifaction of the language.

File: docs/03-computation/02-configuration.mdx [hunk 1]
```diff
@@ -8,38 +8,49 @@ import Legislation from '../_partials/_legislation.mdx';
...
+</Legislation>
 
-- [General information](#configuring-general-information) about your Organization and its' data processing policies (Equivalent to a [RoPA - Record of Processing Activities](https://www.cnil.fr/en/record-processing-activities))
-- Mappings of your data structures to [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
-- Few choices you have to make about when to authenticate the user (before or after they make a Privacy Request), and when to deliver automatic responses and when to wait for validation
+In blindnet DevKit defining a RoPA means configuring the following:
+- [General information](#general-information) about your Organization, its representatives, its server's locations and policies
+- [Selectors](#selectors) tell the PCE which fields in your database schema correspond to which [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories
```
Comment (m4rk055) [inline docs/03-computation/02-configuration.mdx:19]:
```suggestion
- [Selectors](#selectors) - additional [PRIV](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md) Data Categories, subcategories of the existing ones, to describe the data you are processing.
```

Comment (m4rk055) [reply docs/03-computation/02-configuration.mdx:19]: > which fields in your database schema correspond to which Data Categories

This is done in connector, not PCE.

**Review by m4rk055 — CHANGES_REQUESTED** [review]

**Comment by m4rk055** [comment]
@milstan Delete `package-lock.json` file (use `yarn` to install dependencies, not `npm`).

File: docs/03-computation/02-configuration.mdx [hunk 2]
```diff
@@ -59,25 +70,31 @@ Selectors can be set in the PCE using [PUT /configure/selectors](https://stage.c
 
 :::
 
-Sensitive data you capture on a regular base can often be complex to categorize.
+<Legislation>
+Seeting the selectors is important in order to reply to any `TRANSPARENCY.DATA-CATEGORIES` Privacy Request, and comply with several regulations (GDPR Art. 30.1.c, GDPR Art. 15.1.b, CCPA 1798.100.1, CCPA 1798.110.1.1, CCPA 1798.115.1.1, 1798.115.1.2, CCPA 1798.115.1.3). They are also used to indetify particular user data relevant for a particular request and execute operations over the data storage.
```
Comment (m4rk055) [inline docs/03-computation/02-configuration.mdx:74]:
```suggestion
Setting the selectors is important in order to reply to any `TRANSPARENCY.DATA-CATEGORIES` Privacy Request, and comply with several regulations (GDPR Art. 30.1.c, GDPR Art. 15.1.b, CCPA 1798.100.1, CCPA 1798.110.1.1, CCPA 1798.115.1.1, 1798.115.1.2, CCPA 1798.115.1.3). They are also used to idnetify particular user data relevant for a particular request and execute operations over the data storage.
```

File: docs/03-computation/02-configuration.mdx [hunk 3]
```diff
@@ -59,25 +70,31 @@ Selectors can be set in the PCE using [PUT /configure/selectors](https://stage.c
...
+</Legislation>
+
+Sensitive data you capture can often be complex to categorize.
 
 The blindnet devkit gives you a clear [set of data categories](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/dictionary/data-categories/en.data-categories.json) you can use to qualify collected sensitive data for efficient privacy management.
 
-A selector can be defined as a subcategory of any of these data categories, using [dot notation](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#term-dot-notation), to qualify [any captured data](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#data-capture-fragments).
+A selector, corresponding to a particular field in your databse, can be defined as a subcategory of any of these data categories. This is done using [dot notation](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#term-dot-notation).
```
Comment (m4rk055) [inline docs/03-computation/02-configuration.mdx:81]:
```suggestion
A selector, corresponding to a particular field in your database, can be defined as a subcategory of any of these data categories. This is done using [dot notation](https://github.com/blindnet-io/product-management/blob/main/refs/schemas/priv/RFC-PRIV.md#term-dot-notation).
```

File: docs/03-computation/02-configuration.mdx [hunk 4]
```diff
@@ -59,25 +70,31 @@ Selectors can be set in the PCE using [PUT /configure/selectors](https://stage.c
...
 :::caution
 
 Each selector is defined by its full dot notation (`{data_category}.{subcategory}`).
 
 Using just `{subcategory}` will lead to an error.
 
-`{subcategory}` can however be optional, as any data category can be used as a selector.
+`{subcategory}` is however optional, as any data category can be used as a selector.
 
 :::
```
Comment (m4rk055) [inline docs/03-computation/02-configuration.mdx:99]: In that case, the data category already exists and there is no need to define it. This section can be deleted.

**Review by m4rk055 — CHANGES_REQUESTED** [review]

File: docs/03-computation/02-configuration.mdx (hunk 4 above)
Comment (milstan) [reply docs/03-computation/02-configuration.mdx]: True. Deleted

**Review by m4rk055 — APPROVED** [review]

**Pull request merged**